frontend/src/types/common.ts

```diff
@@ -785,6 +807,18 @@ export const empty = (
     },
   };
 
+  const EMPTY_DEPLOYMENT_CONFIG: DeploymentConfig = {
+    id: EMPTY_ID,
+    creator: EMPTY_PRINCIPAL,
+    createdTs: 0,
+    updater: EMPTY_PRINCIPAL,
+    updatedTs: 0,
+    project: EMPTY_PROJECT,
+    schedule: {
+      deployments: [],
```

Comment:
same here

Reply:
This is aligned to https://github.com/bytebase/bytebase/blob/main/api/deploy.go#L35